### DISCUSSION TITLE
Exercise 2: Link to notebook

### DISCUSSION THREAD

**Discussion by buchmann-dhi** [DHI/python-package-development#12] [General]
In assignment no 2, the link to the "notebook", should presumably(?) be a *ipynb file, but it is really a html-file. I wonder what the "download notebook" means.
https://dhi.github.io/python-package-development/projects/data_cleaning/Project_module_02.html
https://dhi.github.io/python-package-development/projects/data_cleaning/notebook_A.html
Further, it seems like we must (or should have) install Jupyter on our machine to actually use the notebook feature.
Could you be explicit about this (I have Jupyter, but others might not even know about it?)

**Comment by buchmann-dhi** [comment]
@ecomodeller and @jsmariegaard: I think we need to get access to the jupyter notebook - not just a html-print - if we should use it for homework(?)

**Reply by ecomodeller** [reply]
Is this the link you are looking for?

https://github.com/DHI/python-package-development/blob/main/projects/data_cleaning/notebook_A.ipynb

**Reply by buchmann-dhi** [reply]
Yes, thank you.
The project assignment links to an html-"print" of the notebook, see above, and to be frank, I had no idea that we should go look for it ourselves under "blob" :P

**Comment by jsmariegaard** [comment]
It happens in the rendering of the site. I have now updated the link in the exercise so it points to the above url.
